lib/reek/spec/should_reek_of.rb

```diff
@@ -8,6 +8,7 @@ module Spec
     # An rspec matcher that matches when the +actual+ has the specified
     # code smell.
     #
+    # :reek:InstanceVariableAssumption
```

Comment:
Yes, every time the matcher is used it needs to re-initialize this value. This is because RSpec allows you to do this:

```ruby
match = reek_of(:UnicornPoo, color: 'pink')
expect(forest).to match
expect(city).not_to match
```